app/templates/resources/create_review.html

```diff
@@ -0,0 +1,47 @@
+{% extends 'layouts/base.html' %}
+{% import 'macros/form_macros.html' as f %}
+
+{% block content %}
+    <div class="ui stackable centered grid container">
+        <div class="twelve wide column">
+            <table class="ui compact definition table">
+                <tr><td>Name</td><td>{{ resource.name }}</td></tr>
+                <tr><td>Description</td><td>{{ resource.description }}</td></tr>
+                <tr><td>Website</td><td>{{ resource.website }}</td></tr>
+                <tr><td>Address</td><td>{{ address.street_address }}</td></tr>
+                <tr><td>User</td><td>{{ user.first_name }} {{ user.last_name }}</td></tr>
+                <tr><td>Reviews</td><td>
+                    {% for review in resource.reviews.all() %}
```

Comment:
let's pass in a reviews variable that is set as `reviews = resource.reviews.all()` so we don't have to query in the templates